### DISCUSSION TITLE
Mobile App Widget Idea

### DISCUSSION THREAD

**Discussion by hybridtracer** [advplyr/audiobookshelf#853] [Ideas]
My previous audiobook app, Listen Audiobook Player, had a really nifty 1x1 widget that was just a book cover and tapping it played and paused the book. It was resizable to any size which made it very handy. If possible a similar feature down the line would be greatly appreciated. Also double tapping opens the app directly to the reader if more input than pause/play is need.

<img src="https://user-images.githubusercontent.com/20684104/180483918-38358f45-46b1-4b0b-9e53-3619dc755eeb.jpg" width="100"> <img src="https://user-images.githubusercontent.com/20684104/180483925-cb7bdf24-7821-4b80-a314-02073ec97c79.jpg" width="100"> <img src="https://user-images.githubusercontent.com/20684104/180483935-d31b479c-385c-463c-9a36-ebbdf67f541b.jpg" width="100"> <img src="https://user-images.githubusercontent.com/20684104/180483940-4bd21f48-4d25-4655-8752-17725403dcda.jpg" width="100">

I attached photos of how it looks at different sizes and how I used it in the last picture. I like having it on the homescreen because it reminds me to listen more. I'm sure it's not a high priority but when widgets are eventually worked on, I would love one that functions like this.

**Comment by advplyr** [comment]
Thanks for the screenshots. I've never built a widget before but played around with it today and got a basic demo working.

**Reply by hybridtracer** [reply]
Thats awesome. this project is shaping out to be incredible already.

**Comment by hpka** [comment]
I'm also in favour of a super simple, clean widget. I suggest the play/pause, back Xsec, forward Xsec buttons alongside the cover art. That's it. The bigger the buttons the better. Tapping the cover art opens the app to the player screen. Similar to what's on the notification shade.
